api/src/main/resources/workbench-api.yaml

```diff
@@ -4767,6 +4767,10 @@ definitions:
         type: boolean
         default: false
         description: Whether user should see a prompt if Workspace Research Purpose needs Review
+      enableRasIdMeLinking:
+        type: boolean
+        default: false
+        description: Whether users are required to link their RAS/ID.me account to AoU
```

Comment:
This is not quite right: users are not **required** to link w/ ID.me - it's a choice of this or Login.gov.  (or at least it will be after RW-10379)